internal/utility/lineposition/json.go

```diff
@@ -0,0 +1,69 @@
+package lineposition
+
+import (
+	"strings"
+
+	"github.com/google/osv-scanner/internal/cachedregexp"
+	"github.com/google/osv-scanner/pkg/models"
+)
+
+func InJSON[P models.ILinePosition](groupKey string, dependencies map[string]P, lines []string, offset int) {
```

Comment:
This is a pretty long method, can we split it in multiple ones ?

Reply:
Yes, let's do it in https://github.com/DataDog/osv-scanner/pull/27 if that's okay?